### PR TITLE
Update data documentation and some data steps

### PR DESCRIPTION
**Pull request by switzersc-usds**
I ran through the score comparison workflow to ensure it works locally and make sure it's documented sufficiently for external contributions and use. This PR adds documentation and updates some of the steps for ETL and score comparison so that it runs smoothly.

### CODE REVIEW

**Review by esfoobar-usds — DISMISSED** [review]
🍏

**Comment by switzersc-usds** [comment]
@esfoobar-usds @lucasmbrown-usds could either of y'all please leave another review and then either update if it needs an update or merge if approved? Thanks!

File: data/data-pipeline/README.md [hunk 1]
```diff
@@ -74,7 +74,31 @@ _For example: `poetry run python application.py etl-run ejscreen` would only run
...
+  pip install dynaconf
+  pip instal xlsxwriter
+```
+1. Start the notebooks: `jupyter notebook`
+1. In your browser, navigate to one of the URLs returned by the above command.
+1. Select `scoring_comparison.ipynb` from the options in your browser.
+1. Run through the steps in the notebook. You can step through them one at a time by clicking the "Run" button for each cell, or open the "Cell" menu and click "Run all" to run them all at once.
+1. Reports and spreadsheets generated by the comparison tool will be available in `data/data-pipeline/data/comparison_outputs`.
+
+*NOTE:* This may take several minutes or over an hour to fully execute and generate the reports.
```
Comment (ginabeena) [inline data/data-pipeline/README.md:101]: Nice work on documenting this @switzersc-usds !!

**Review by ginabeena — APPROVED** [review]
:shipit:

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Pull request merged**